website/content/docs/installation.md

```diff
@@ -1,83 +1,195 @@
 +++
 description = "How to install Bloop on your machine"
-title = "The installation process"
+title = "Installation"
 date = "2018-02-08T16:35:00+01:00"
 draft = false
 weight = 1
 bref = "Here are all the instructions to install Bloop"
 toc = true
 +++
 
-### Instructions for Mac OS
+### Mac OS
 
-Installing Bloop with [Homebrew](https://brew.sh) is the easiest and fastest way to install Bloop on
-Mac OS.
-
-We publish an updated formula along with every release of Bloop. You can install it using:
+Install with [Homebrew](https://brew.sh) via:
 
 ```sh
 $ brew install scalacenter/bloop/bloop
 ```
 
-The formula will also take care of installing a property list (`plist`) that will take care of
-starting the Bloop server when you log onto your machine. This way, you don't have to think about
-starting it yourself.
+The formula installs a property list (`plist`) that starts the Bloop server
```

Comment:
Just checking: is "formula" specific Brew terminology? Sounds strange, if not.

Reply:
Yes, it is :smile:

Reply:
This is how they call package definition in Homebrew world